docker/gitpod/Dockerfile

```diff
@@ -4,10 +4,15 @@ LABEL org.opencontainers.image.source=https://github.com/freecodecamp/freecodeca
   org.opencontainers.image.description="A Gitpod image for the main freeCodeCamp repository" \
   org.opencontainers.image.licenses=BSD-3-Clause
 
-# from https://www.gitpod.io/docs/introduction/languages/javascript#node-versions
-RUN bash -c 'VERSION="20" \
-  && source $HOME/.nvm/nvm.sh && nvm install $VERSION \
-  && nvm use $VERSION && nvm alias default $VERSION \
-  && npm i -g pnpm@9 \
-  && echo "nvm use default &>/dev/null" >> ~/.bashrc.d/51-nvm-fix \
-  && pnpm dlx playwright@1.47.1 install --with-deps chromium'
+# Install Node.js, pnpm, and Playwright with specific versions
+RUN bash -c '\
+    VERSION="20" \
+    && source $HOME/.nvm/nvm.sh \
+    && nvm install $VERSION \
+    && nvm use $VERSION \
+    && nvm alias default $VERSION \
+    && npm i -g pnpm@9 \
+    && echo "nvm use default &>/dev/null" >> ~/.bashrc.d/51-nvm-fix \
+    && PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 pnpm dlx playwright@1.47.1 install \
+    && pnpm dlx playwright@1.47.1 install --with-deps chromium \
```

Comment:
Why do we need to install playwright twice?

Reply:
The first command installs Playwright without downloading any browsers, allowing for the library to be used with pre-installed browsers. The second command specifically installs Chromium and its dependencies, ensuring that the necessary browser is available for testing. This approach provides flexibility in managing installations while avoiding unnecessary downloads.

Reply:
But it doesn't? It's literally an extra install step for no reason.

Reply:
both of them serves different purposes, its not the same
`PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 pnpm dlx playwright@1.47.1 install`


This first installation skips browser download (due to the environment variable)
It installs only the Playwright framework/library itself

`pnpm dlx playwright@1.47.1 install --with-deps chromium`

This second command specifically installs Chromium browser and its dependencies
The --with-deps flag ensures all necessary dependencies for Chromium are installed